docs/source/excel_submission.rst

```diff
@@ -0,0 +1,362 @@
+.. role:: raw-html-m2r(raw)
```

Comment:
Looks like a lot of information in this file especially that needs to be redone for smaht, particularly from lines 145 downward which refers to 4DN-specific tools/ideas. If we're not going to show this page, then might be fine to leave in with a TODO about updating these.